tests/unit/cli/test_main.py

```diff
@@ -26,6 +27,20 @@ def test_cli_some_command(self):
             result = runner.invoke(cli, ["local", "generate-event", "s3"])
             self.assertEqual(result.exit_code, 0)
 
+    def test_cli_with_no_region_arg_validate(self):
+        mock_cfg = Mock()
+        with patch("samcli.cli.main.GlobalConfig", mock_cfg):
+            runner = CliRunner()
+            runner.invoke(cli, ["validate", "--region", "--debug"])
+            self.assertRaises(RegionError)
+
+    def test_cli_with_no_region_arg_deploy(self):
```

Comment:
I don't quite get this test.

why does deploy use use-container False?

Reply:
Hmm I took the parameters from a separate test, but ultimately I just want to test the region behaviour. Would it work with "deploy --region --debug"? If so, I will just remove the use-container parameter